domains/markaguirre.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Developer Portfolio",
+    "repo": "https://github.com/MarkAguirre26/folio",
+    "owner": {
+        "username": "MarkAguirre26",
+        "email": "markaguirre26@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://markaguirre.netlify.app/"
```

CNAME should not contain protocol
```suggestion
        "CNAME": "markaguirre.netlify.app"
```